samples/media/video/src/main/java/com/example/platform/media/video/TransformerVideoComposition.kt

```diff
@@ -173,12 +190,13 @@ class TransformerVideoComposition : Fragment() {
             // sequence.
             val audioSequence = EditedMediaItemSequence(
                 ImmutableList.of(backgroundAudio),
-                /* isLooping*/true
+                /* isLooping*/true,
             )
             compositionSequences.add(audioSequence)
         }
 
-        return Composition.Builder(compositionSequences).build()
+        return Composition.Builder(compositionSequences)
+            .setVideoCompositorSettings(VideoCompositorSettings.DEFAULT).build()
```

Comment:
from @droid-girl , adding ` .experimentalSetForceAudioTrack(true)` here resolves the issue. Although this seems to only be needed when an image is the first item in the composition.